LEAF_Request_Portal/admin/templates/mod_templates_reports.tpl

```diff
@@ -1,634 +1,20 @@
 <link rel=stylesheet href="../../libs/js/codemirror/addon/merge/merge.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.62.2/theme/lucario.min.css">

```

Comment:
Ditto

Reply:
I didn't notice that the theme was available locally. The Library has been changed.